model/registry/event.yaml

```diff
@@ -16,3 +16,13 @@ groups:
           separation of semantics for events in separate domains like browser, mobile, and
           kubernetes.
         examples: ['browser.mouse.click', 'device.app.lifecycle']
+      - id: message
```

Comment:
maybe `event.description`?

Reply:
I took `message` as it's a convention that already exists... but granted it could be confusing with messaging.: `description` is not bad idea.